src/Swift.Bindings/src/Model/ModuleParsingResult.cs

```diff
@@ -1,7 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
 namespace BindingsGeneration
 {
     public sealed record ModuleParsingResult(
         ModuleDecl ModuleDecl,
-        Dictionary<NamedTypeSpec, TypeDecl> TypeDecls
+        Dictionary<NamedTypeSpec, TypeDecl> TypeDecls,
+        List<NamedTypeSpec> ClosedGenericTypes
```

Comment:
What does "Closed" mean in this context?

Reply:
I meant generic types which have all of their arguments specified. We need to calculate size, stride, alignment and frozenness for those

Reply:
Thanks - can I instead suggest the term "bound" which I've seen used more often?